README.md

```diff
@@ -1,6 +1,74 @@
 # pxt-rovercode
 
+## Development Usage
 
+```bash
+$ sudo npm install -g pxt
```

Comment:
This needs to be installed globally or can it just be in the local directory as a dev dependency? The next commands could then be scripts in `pxt.json`?

Reply:
Are you thinking we would add a package.json that lists pxt as a dev dependency, so you can run `npm install` to get pxt?

Reply:
I just tried it, and the `pxt` command doesn't seem to recognize scripts in pxt.json.
So I think our only option would be to put the script in a package.json. Then we could abstract away the entire pxt interface. That'd probably be better?

Reply:
I think so. Then it would be standard with what other projects do. I'm not sure why there's a `pxt.json` that looks exactly like a `package.json`. It even has a `package-lock.json`

Reply:
Done.